bundles/org.openhab.binding.sonyprojector/README.md

```diff
@@ -187,6 +187,129 @@ The following channels are available:
+| COLOR_SPACE         | Color space               |
+| COLOR_TEMP          | Color temperature         |
+| GAMMA_CORRECTION    | Gamma correction          |
+| IRIS_MODE           | Iris mode                 |
+| COLOR_CORRECTION    | Color correction          |
+| PITCH               | Screen pitch              |
+| SHIFT               | Screen shift              |
+| ASPECT              | Aspect mode               |
+| ASPECT_Normal       | Aspect normal             |
+| ASPECT_Full         | Aspect full               |
+| ASPECT_Full1        | Aspect full 1             |
+| ASPECT_Full2        | Aspect full 2             |
+| ASPECT_Zoom         | Aspect zoom               |
+| ASPECT_WideZoom     | Aspect wide zoom          |
+| ASPECT_VStretch     | Aspect V stretch          |
+| ASPECT_Stretch      | Aspect stretch            |
+| ASPECT_Squeeze      | Aspect squeeze            |
+| ASPECT_185          | Aspect 1.85:1 zoom        |
+| ASPECT_235          | Aspect 2.35:1 zoom        |
+| ASPECT_Anamorphic   | Aspect anamorphic         |
+| ASPECT_Subtitle     | Aspect subtitle           |
+| APA                 | APA                       |
+| DOT_PHASE           | Dot phase                 |
+| V_KEYSTONE          | V keystone                |
+| V_KEYSTONE_UP       | V keystone +              |
+| V_KEYSTONE_DOWN     | V keystone -              |
+| LENS_CONTROL        | Lens control              |
+| LENS_SHIFT          | Lens shit                 |
+| LENS_SHIFT_LEFT     | Lens shit LEFT            |
+| LENS_SHIFT_RIGHT    | Lens shit RIGHT           |
+| LENS_SHIFT_UP       | Lens shit UP              |
+| LENS_SHIFT_DOWN     | Lens shit DOWN            |
+| LENS_ZOOM           | Lens zoom                 |
+| LENS_ZOOM_LARGE     | Lens zoom large           |
+| LENS_ZOOM_SMALL     | Lens zoom small           |
+| LENS_FOCUS          | Lens focus                |
+| LENS_FOCUS_FAR      | Lens focus far            |
+| LENS_FOCUS_NEAR     | Lens focus near           |
+| LENS_POSITION       | Lens position             |
+| PICTURE_POSITION    | Picture position          |
+| PICTURE_POS_185     | Picture position 1.85:1   |
+| PICTURE_POS_235     | Picture position 2.35:1   |
+| PICTURE_POS_CUSTOM1 | Picture position custom 1 |
+| PICTURE_POS_CUSTOM2 | Picture position custom 2 |
+| PICTURE_POS_CUSTOM3 | Picture position custom 3 |
+| MODE_3D             | 3D                        |
+| OPTIONS             | Options                   |
+| EXIT                | Exit                      |
+| SYNC_MENU           | Sync menu                 |
+| PLAY                | Play                      |
+| STOP                | Stop                      |
+| PAUSE               | Pause                     |
+| FAST_REWIND(        | Fast rewind               |
+| FAST_FORWARD        | Fast forward              |
+| PREVIOUS            | Previous                  |
+| NEXT                | Next                      |
+
```

Comment:
For several above, the second word should be capitalized (e.g Fast rewind to Fast Rewind).